src/engine/loopingcontrol.cpp

```diff
@@ -581,6 +589,19 @@ void LoopingControl::slotReloopToggle(double val) {
     }
 }
 
+void LoopingControl::slotReloopCue(double pressed) {
+    if (pressed > 0) {
+        QMutexLocker lock(&m_mutex);
```

Comment:
No, since this is the only place it is used. 
It cannot be a concurrency issue here, because we do not read data.